api/src/main/java/jakarta/resource/cci/ResourceWarning.java

```diff
@@ -84,6 +84,7 @@ public ResourceWarning(String message, String errorCode) {
      * <code>initCause</code> methods of the 
      * <code>java.lang.Throwable</code> class.
      */
+    @Deprecated
```

Comment:
When deprecating, you should tell the user what you expect them to use instead.

Reply:
I do not deprecate anything, see `@javadoc` in javadoc. What do you want the person who just follows compiler advice to add there?